0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -150,11 +153,12 @@ We assume that the SPSP Client knows the SPSP Server's SPSP Endpoint (see [Payme
 
 1. The user's SPSP Client queries the SPSP Server's SPSP Endpoint.
 
-2. The SPSP Endpoint responds with the SPSP Server's info, namely the ILP address (`destination_account`) and the shared secret (`shared_secret`) to be used in STREAM. 
-    * The `destination_account` MUST be used as the STREAM `destinationAccount`.
-    * The `shared_secret` MUST be decoded from base64 and used as the STREAM `sharedSecret`.
-  
-    It MAY respond with additional information if it is an invoice server or a pull payment server. For more information, see [SPSP Invoices](../0037-spsp-invoices/0037-spsp-invoices.md) and [SPSP Pull Payments](../0036-spsp-pull-payments/0036-spsp-pull-payments.md).
+2. The SPSP Endpoint responds with the SPSP Server's info, namely the ILP address (`destination_account`) and the shared secret (`shared_secret`) to be used in STREAM.
+
+   - The `destination_account` MUST be used as the STREAM `destinationAccount`.
+   - The `shared_secret` MUST be decoded from base64 and used as the STREAM `sharedSecret`.
+
+   It MAY respond with additional information if it is an invoice server or a pull payment server. For more information, see [SPSP Invoices](../deprecated/0037-spsp-invoices/0037-spsp-invoices.md) and [SPSP Pull Payments](../deprecated/0036-spsp-pull-payments/0036-spsp-pull-payments.md).
```

Comment:
Apologies for the other Prettier related changes, but these URLs are the crux of the update.